bin/ruby-build

```diff
@@ -1285,8 +1285,8 @@ build_package_enable_yjit() {
 
   local rustc_ver="$(rustc --version 2>/dev/null)"
   [ -n "$rustc_ver" ] || return 0
-  # Some kind of built-in bash comparison for dotted version strings would be awesome.
-  if [[ "${rustc_ver}" == *"rustc 1."[6789]* ]]; then
+  # YJIT supports Rust 1.58.1 and later
+  if [[ "${rustc_ver}" =~ ^rustc\ 1\.(58\.1|59\.|[6789]\d\.) ]]; then
```

Comment:
Should it accept 1.58.2 till 1.58.9?

Reply:
I don't think there are releases with those numbers: https://github.com/rust-lang/rust/blob/master/RELEASES.md